scripts/prepare.ts

```diff
@@ -103,12 +110,39 @@ async function tryReadConfigFile(): Promise<string> {
   return configFile.toString();
 }
 
+async function readPermissionsUseCasesFile(): Promise<string> {
+  log(
+    `Reading permissions use cases file from path '${permissionsUseCasesPath}'.`
+  );
+
+  const useCasesFileContents = await fs.readFile(permissionsUseCasesPath);
+  const useCases = yaml.load(useCasesFileContents.toString()) as Record<
+    string,
+    any
+  >;
+
+  const useCasesObject = {
+    'permissions-use-cases-json': JSON.stringify(
+      useCases['useCases'],
+      replacer
+    ),
+  };
+
+  log('Read permissions use cases file successfully.');
+
+  return yaml.dump(useCasesObject);
```

Comment:
This code duplicates the one from `getPermissionsUseCasesConfig` function. Is there a reason for that?

Reply:
Not a good one 😬 they were different at the beginning and I forgot to refactor. Will consolidate them into one